### PR TITLE
Updated simset_sim

### PR DESCRIPTION
**Pull request by txusser**
Now, second sample simulation (for estimating n_photons) will only be performed if importance sampling is activated. This solves a problem when the user does not want to use importance sampling.

Additionally, I entered a condition that will only trigger this calculation if the user has not stated photons manually.

So, correct way of launching simulations with importance sampling and automatic calculations should be:

s_photons = 1543214 (int num)
photons = 0

This is well matched with photons = 0  and s_photons=0 triggers autocalculation (from simset) and no importance sampling

### CODE REVIEW

**Comment by txusser** [comment]
@YerePhy this has been sitting here for a while

File: simpet.py [hunk 1]
```diff
@@ -1,4 +1,5 @@
 import os
+
```
Comment (YerePhy) [inline simpet.py]: Tiene alguna finalidad xD?

File: src/simset/simset_sim.py [hunk 2]
```diff
@@ -93,19 +102,17 @@ def run(self):
...
 
-        print(" ")
 
         for process in processes:
             process.join()
 
         print(" ")
 
-        self.simulation_postprocessing()
+        #self.simulation_postprocessing()
```
Comment (YerePhy) [inline src/simset/simset_sim.py]: Aquí no se rompe nada seguro? Si la linea va a quedarse comentada mejor borrarla

File: src/simset/simset_tools.py [hunk 3]
```diff
@@ -493,6 +493,7 @@ def make_index_file(simulation_dir, simset_dir, log_file=False):
 
 
 def process_weights(weights_file, output_dir, scanner, add_randoms=0):
+
```
Comment (YerePhy) [inline src/simset/simset_tools.py]: Alguna finalidad? O es por algún formato en específico?

File: simpet.py (hunk 1 above)
Comment (txusser) [reply simpet.py]: No, probablemente el autoformater le pareció una buena idea jaja

File: src/simset/simset_sim.py (hunk 2 above)
Comment (txusser) [reply src/simset/simset_sim.py]: Este comentario HAY que quitarlo, se me debió colar en el último commit que comenté esta línea para ver las salidas sin postprocessing. Lo arreglo

File: src/simset/simset_tools.py (hunk 3 above)
Comment (txusser) [reply src/simset/simset_tools.py]: Creo que es el formateador automático de JetBrains que le deben gustar los espacios

**Review by txusser — COMMENTED** [review]
All the small things that we found have been solved

**Comment by txusser** [comment]
@YerePhy I think that this is good to go. In any case, may you make some simulations from your processing on these branch to check that everything keeps going as expected?

**Pull request merged**